metro.sh

```diff
@@ -48,7 +48,7 @@ for s in $STATION; do
                  </soap:Envelope>')"
 
     # Print output
-    echo -e "\n* Estación: ${STATION_NAME}"
+    echo -e "\n* Station: ${STATION_NAME}"
```

Comment:
This is not necessary. This script is intended to be used mainly by Spanish speakers.